packages/terra-clinical-result/src/common/utils.js

```diff
@@ -35,17 +71,10 @@ const checkTypeNumeric = (resultData) => {
  */
 const ConditionalWrapper = ({ condition, wrapper, children }) => (condition ? wrapper(children) : children);
 
-const ClinicalResultUtils = {
-  isEmpty,
-  checkIsStatusInError,
-  checkTypeNumeric,
-  ConditionalWrapper,
-};
-
-export default ClinicalResultUtils;
```

Comment:
Removing this default export is technically non-passive unless we've explicitly marked this file as private. We have taken to creating  a `/private` directory in other repos to explicitly mark files as not for consumer use and to give us leeway to make changes like this.

I don't anticipate that we have any direct consumer usage of this file, so I bring this up more as an FYI than a call for changes. But it's important for us to think about what APIs we expose from our packages, because often it's more than just the component definition.

Reply:
Since it is a MVB, I have reverted the changes and will put the change in another PR https://github.com/cerner/terra-clinical/pull/648/commits/b866ddd2f17961f96d98cd25553d8f0254d61d11